packages/at_auth/lib/src/keys/at_keys_io_impl.dart

```diff
@@ -52,5 +50,40 @@ class FileAtKeysIo extends WrittenAtKeysIo {
   }
 }
 
-//TODO: future add simAtKeysIo
-// class SimAtKeysIo extends GeneratedAtKeysIo with KeyIOMixin {
+/// Taken from at_cli_commons, but I can't import it due to circular dependencies
+/// at_cli_commons depends on at_onboarding_cli....
```

Comment:
This shouldn't be a dart doc comment fwiw (dart doc is triple instead of double slash)